http://bugs.gentoo.org/233475#c9
Fix build issue with USE=wxwindows
Patch from upstream (did not make it into gnuplot-4.2.5)

--- configure.in	8 May 2007 22:42:53 -0000	1.219
+++ configure.in	16 Jul 2007 21:58:43 -0000	1.220
@@ -853,6 +853,10 @@
   cp ${srcdir}/tutorial/eg3.dat tutorial
 fi
 
+dnl explicit call to PKG_PROG_PKG_CONFIG because the first call to
+dnl PKG_CHECK_MODULES may not happen
+PKG_PROG_PKG_CONFIG
+
 dnl  wxWidgets terminal
 
 dnl wxWidgets terminal needs C++
